react/features/base/tracks/middleware.web.ts

```diff
@@ -113,9 +112,7 @@ MiddlewareRegistry.register(store => next => action => {
                 APP.UI.setVideoMuted(participantID);
             }
         } else if (jitsiTrack.isLocal()) {
-            APP.conference.setAudioMuteStatus(muted);
-        } else {
-            APP.UI.setAudioMuted(participantID, muted);
+            APP.conference.updateAudioIconEnabled();
```

Comment:
Same question here again. If we already know that a local audio track exists, why do we need to check if the mic icon needs to be enabled or disabled?